rcl/include/rcl/graph.h

```diff
@@ -180,6 +180,51 @@ rcl_get_service_names_and_types_by_node(
   const char * node_namespace,
   rcl_names_and_types_t * service_names_and_types);
 
+/// Return a list of service client names and types associated with a node.
+/**
+ * The `node` parameter must point to a valid node.
+ *
+ * The `service_names_and_types` parameter must be allocated and zero initialized.
+ * This function allocates memory for the returned list of names and types and so it is the
+ * callers responsibility to pass `service_names_and_types` to rcl_names_and_types_fini()
+ * when it is no longer needed.
+ * Failing to do so will result in leaked memory.
+ *
+ * \see rcl_get_publisher_names_and_types_by_node for details on the `no_demangle` parameter.
+ *
+ * The returned names are not automatically remapped by this function.
+ * Attempting to create clients using names returned by this function may not
```

Comment:
Should this be "attempting to create service servers"?

Reply:
And vice-versa on the other doc?

Reply:
Yeah that makes sense. Updated in da774b1